### PR TITLE
[mod_banners]XML code style

### PR DESCRIPTION
**Pull request by wojsmol**
### Summary of Changes

Code style changes for mod_banners
### Testing Instructions

code review

cc @JoomliC @andrepereiradasilva

### CODE REVIEW

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on b9622c1ee41544955895bcdee6f39c41ed57a808<br /><br />on code review<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10577">issues.joomla.org/joomla-cms/10577</a>.</sub>

File: modules/mod_banners/mod_banners.xml [hunk 1]
```diff
@@ -21,30 +21,33 @@
...
+					label="MOD_BANNERS_FIELD_COUNT_LABEL"
+					description="MOD_BANNERS_FIELD_COUNT_DESC"
 					default="5"
 					class="validate-numeric"
 					filter="integer"
-					label="MOD_BANNERS_FIELD_COUNT_LABEL"
-					description="MOD_BANNERS_FIELD_COUNT_DESC" />
+				/>
+
 				<field
```
Comment (JoomliC) [inline modules/mod_banners/mod_banners.xml]: <code>/></code> is not in a new line for this field...

Comment (wojsmol) [inline modules/mod_banners/mod_banners.xml]: this field os ok, but one below not

**Comment by wojsmol** [comment]
@JoomliC I updated this PR.

**Comment by andrepereiradasilva** [comment]
I have tested this item :white_check_mark: successfully on d186e14fd34b6e3109c72f1db5de7074fd2f8935<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10577">issues.joomla.org/joomla-cms/10577</a>.</sub>

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on d186e14fd34b6e3109c72f1db5de7074fd2f8935<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10577">issues.joomla.org/joomla-cms/10577</a>.</sub>

**Comment by brianteeman** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/10577">issues.joomla.org/joomla-cms/10577</a>.</sub>

**Pull request closed without merging**